config/filter.d/nextcloud-auth.conf

```diff
@@ -0,0 +1,18 @@
+# Fail2Ban filter file for Nextcloud login failures
+#
+# Author: Eric Wolf
+#
+
+[INCLUDES]
+
+# Read common prefixes
+before = common.conf
+
+after = nextcloud-auth.local
```

Comment:
Including `common.conf` is required since it provides `__prefix_line`.
I added the `%(__prefix_line)s?` part to allow using it when using the systemd journal for logging since it would not match without it.
The [example](https://fail2ban.readthedocs.io/en/latest/filters.html#filter-file) in the filter guide made me assume that the `after` include was required, turns out I was wrong.